pkg/importer/util.go

```diff
@@ -38,10 +40,16 @@ func MoveFile(src, dst string) error {
 	return err
 }
 
-// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file
-func StreamDataToFile(dataReader io.Reader, filePath string) error {
+// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file or block device
+func StreamDataToFile(dataReader io.Reader, filePath, volumeMode string) error {
```

Comment:
volumeMode should not be a string, it has a type.

Reply:
changed according to your comment above.